server/lib/chat/chat-service.ts

```diff
@@ -1128,45 +1150,15 @@ export default class ChatService {
         action: 'userActive2',
         userId: userSockets.userId,
       })
-      userSockets.subscribe(getChannelPath(userChannel.channelId))
+      userSockets.subscribe<ChatInitActiveUsersEvent>(
+        getChannelPath(userChannel.channelId),
+        () => ({
+          action: 'initActiveUsers',
+          activeUserIds: this.state.channels.get(userChannel.channelId)!.toArray(),
```

Comment:
Same as above

Reply:
And perhaps these 2 subscribes should be pulled out into a common method or something so this code isn't duplicated?